orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/general/EqualTo.feature

```diff
@@ -123,18 +132,21 @@ Feature: User can specify that a value is equalTo a required value
       | foo |
       | 15  |
 
-  Scenario: 'EqualTo' a value and must be null should be contradictory
-    Given there is a non nullable field foo
+  Scenario: 'EqualTo' a value and must be null should permit a or null
```

Comment:
Should this really say "and must be null"?  Would "and can be null" be more correct?